pagespeed/apache/apache_rewrite_driver_factory.cc

```diff
@@ -227,4 +236,18 @@ ApacheConfig* ApacheRewriteDriverFactory::NewRewriteOptionsForQuery() {
   return new ApacheConfig("query", thread_system());
 }
 
+GoogleString ApacheRewriteDriverFactory::MasterPidFilename() {
```

Comment:
It seems like you use 'master' here for the same thing a lot of the code uses 'root' for, maybe worth considering using the same terminology?